webapp/src/store/ProjectStore.ts

```diff
@@ -62,9 +73,12 @@ export class ProjectStore {
     this.data.languages[lang][id].text = text;
   }
 
-  private async initIfNecessary() {
-    if (Object.keys(this.data.languages).length == 0) {
-      this.data.languages = await this.translationAdapter.getTranslations();
-    }
+  private async refresh() {
+    const fromRepo: StoreData = {
+      languages: await this.translationAdapter.getTranslations(),
```

Comment:
It's not safe to read translations from the filesystem while there can be concurrent writes, either by `createPullRequest` or by git via `cloneIfNotExist` and `checkoutBaseAndPull`.

We have the problem on `main` too but its a little milder there as we only read if `languages` is empty.

Let's create a separate concurrency issue and accept a partial regression meanwhile.